### DISCUSSION TITLE
Debug Log When Starting App

### DISCUSSION THREAD

**Discussion by gabrielgasp** [Papooch/nestjs-cls#74] [Q&A]
First of all, huge thanks for this library, made my life a whole lot easier!

I couldn't help but notice that when starting the app, there is a DEBUG log when the ClsMiddleware is mounted. Is there an option to change this to a regular log (like the ones Nest uses)?

Ps: this doesn't affect the usability of the library, just my OCD being triggered 😆

![Screenshot 2023-06-20 at 22 02 42](https://github.com/Papooch/nestjs-cls/assets/83838669/2e0a3814-0cd9-40e5-96c4-9725dd36df40)

**Comment by Papooch** [comment]
It follows the log level set by the NestJS logger, you can set it to 'info' to suppress it.

Although, I might change it to only show when the `DEBUG` env. variable is set, so it better aligns with how NestJS debug logging work 🤔

**Reply by gabrielgasp** [reply]
I think you can either make it conditional based on the `DEBUG` env variable, or just change the logging method to use the regular `log` method.

I actually enjoy having the middleware mount announcement during app init, so I would probably go with option 2, but either one is fine.
